server/dataone_package.py

```diff
@@ -101,18 +144,22 @@ def get_file_md5(file_object, md5):
         return None
     handle.close()
```

Comment:
I think Python supports a `finally` block in its implementation of exception handling so maybe you can squash those two calls to `handle.close()` into one.